### DISCUSSION TITLE
如何让缩略语对照表中的内容表格在页面中顶格显示

### DISCUSSION THREAD

**Discussion by earlstart-1** [sjtug/SJTUThesis#1023] [提问] [answered]
如上描述的，通过这个模板，我无法直接通过表格内置的[!htb]等命令让表格在页面中置顶。图1为缩略语的首页，图二为第二页。
![图1](https://github.com/sjtug/SJTUThesis/assets/163672211/833297f7-18a3-4fbf-9ed1-ad31997d81a9)
![图2](https://github.com/sjtug/SJTUThesis/assets/163672211/83b29a3a-caf6-4740-a39c-afab91f0eb27)

除此之外，如何将摘要、图片索引等内容插入目录页。这个问题，我也没有解决。求大佬帮助！
![图3](https://github.com/sjtug/SJTUThesis/assets/163672211/5b53cf3a-30e1-4cc3-810d-617e64006c86)

**Comment by LogCreative** [comment, marked as answer]
如果你的缩略语对照表是通过常规表格实现的，尝试对照[例子](https://github.com/sjtug/SJTUThesis/blob/0eff2175eb573293a1d274262a2325c6ab5c41c0/contents/floats.tex#L192-L260)重新构造跨页表格。

**Reply by LogCreative** [reply]
当然，如果你是分两个表格实现的，可以考虑后一个表格直接用 [t] 选项（Top）。

**Reply by earlstart-1** [reply]
感谢，第一个方法目测是可以的，但是还存在一些小问题，但是整体思路可以。分两个表格构造第二个表格无法通过[t]直接实现第二个表格置顶。谢谢！

**Comment by LogCreative** [comment]
一般而言，摘要、目录、插图索引等不会放入目录。如果确有必要，考虑删除下面命令中的 `*` 将插图索引、表格索引加入目录：

https://github.com/sjtug/SJTUThesis/blob/0eff2175eb573293a1d274262a2325c6ab5c41c0/main.tex#L39-L43

如果需要完全跟图中一致，可以再使用 SJTUTeX 2.1 版本，在 [Releases](https://github.com/sjtug/SJTUTeX/releases/tag/v2.1) 中下载 `sjtutex.tds.zip` 并覆盖到 `texmf/` 文件夹中。

**Reply by AlexaraWu** [reply]
可以下载 v2-dev 分支的压缩包。
